src/main/java/org/dita/dost/reader/GenListModuleReader.java

```diff
@@ -405,7 +405,8 @@ public void startElement(final String uri, final String localName, final String
 
         final URI href = toURI(atts.getValue(ATTRIBUTE_NAME_HREF));
         final String scope = atts.getValue(ATTRIBUTE_NAME_SCOPE);
-        if (href != null && !ATTR_SCOPE_VALUE_EXTERNAL.equals(scope) && !ATTR_SCOPE_VALUE_PEER.equals(scope)) {
+        if (href != null && !href.toString().startsWith("#") &&
```

Comment:
Instead of `!href.toString().startsWith("#")` this should be e.g. `href.getPath() != null`